setup.py

```diff
@@ -5,9 +5,9 @@
     long_description = readme.read()
 
 setup(
-    name='edx-ecommerce-api-client',
+    name='edx-rest-api-client',
     version='1.1.2',
-    description='Slumber client used to access APIs exposed by the edX E-Commerce Service',
+    description='Slumber client used to access various edX Platform REST APIs.',
```

Comment:
@mattdrayer given that use of this API client is dictated entirely by [Slumber's conventions](http://slumber.readthedocs.org/en/v0.6.0/tutorial.html#using), I think calling this out is a good idea.